### DISCUSSION TITLE
How can obtain the value of .getDistance() in the orbitControls?

### DISCUSSION THREAD

**Discussion by Sea-DH1** [pmndrs/drei#508] [Q&A]
I did not look at this method on the properties of the OrbitControls component

![image](https://user-images.githubusercontent.com/26751491/128327350-5ebd04c1-6a9b-4193-8627-6fc40a141332.png)

**Comment by joshuaellis** [comment]
This looks like the THREE object for OrbitControls, are you sure its a method of said example?
